src/components/axes/axis.ts

```diff
@@ -283,15 +279,13 @@ module Plottable {
     }
 
     /**
-     * Gets the current tick mark length.
-     *
-     * @returns {number} the current tick mark length.
+     * Gets the current tick mark length in pixels.
```

Comment:
"mark" is necessary, since "tick" refers to more than the mark.

I tried to have "in pixels" everywhere that a measurement was in pixels.